src/styles/components/_commander-option.scss

```diff
@@ -1,12 +1,24 @@
 .c-commander-option {
     border: 0;
     padding: 0;
-    background-color: $color-black;
-    color: $color-white;
+    background-color: rgb(51 38 1 / 17%);
```

Comment:
$color-primary with opacity applied, could variable-ize.

Reply:
I'd definitely prefer this to be variable-ized.  Could you actually apply the opacity here and change the color to `#D6D1C7`?